src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Class/Enum name should be in singular form
```suggestion
public enum Color {

```